thermalnetwork/network.py

```diff
@@ -670,15 +674,17 @@ def update_sys_params(self, system_parameter_path: Path, output_directory_path:
         # Load the existing system parameters
         sys_params: dict = json.loads(system_parameter_path.read_text())
         ghe_params: list = sys_params["district_system"]["fifth_generation"]["ghe_parameters"]["ghe_specific_params"]
-        for ghe_id in output_directory_path.iterdir():
-            # each ghe_id is a directory
-            if not ghe_id.is_dir():
-                continue
```

Comment:
Can we keep this check in? There are sometimes other files that we want to skip over during this loop